docs/Using Price Feeds/get-the-latest-price.html

```diff
@@ -17,12 +17,20 @@
 - **EVM-compatible networks**
 - [Solana network](../get-the-latest-price-solana/)
 
-You can use smart contracts to get asset prices on EVM-compatible networks. These examples show you how to get the price of Ethereum (ETH) on the Ethereum network, but you can modify the examples to work on other EVM-compatible networks as well. The list of price feeds for each network are available on the [Price Feed Contracts](../reference-contracts/) page. For development, use testnet feeds such as the [ETH/USD Price Feed](https://kovan.etherscan.io/address/0x9326BFA02ADD2366b30bacB125260Af641031331) on the Kovan testnet.
+To get a full list of supported blockchains for data feeds, see the [Data Feeds Contract Addresses](../reference-contracts/) page.
+
+> 📘 Note
+>
+> The complete list of data feeds for each network are available on the [Data Feed Contracts](../reference-contracts/) page.
 
 > 📘 New Feed Registry
-> You can use the [Feed Registry](../feed-registry/) to reference price feed assets by name or currency identifier instead of by pair/proxy address.
+> You can now use the [Feed Registry](../feed-registry/) to reference data feed assets by name or currency identifier instead of pair/proxy address.
```

Comment:
```suggestion
> You can use the [Feed Registry](../feed-registry/) to reference price feed assets by name or currency identifier instead of by pair/proxy address.
```